apache/kafka
# KAFKA-8741: optimize the use of mx4j web interface

1. Since mx4j-tools.jar is not packaged into the final released tgz file by default, we can't monitor kafka brokers by web inferface. If we want to do it, we must find a mx4j-tools.jar in the maven repository by our own and put it under the directory $KAFKA_HOME/libs, which is very inconvenient.
2. In Mx4jLoader, we read the property values of 'kafka_mx4jenable' and 'mx4jport' from system properties, but there is no way for us to set the switch and customize the port of mx4j-tools. We could add a configuration item in kafka-run-class.sh. For example, if MX4J_PORT is set, we add -Dkafka_mx4jenable=true -Dmx4jport=$MX4J_PORT into the start command.

testing strategy:
1. build the release tar.gz file
2. check if mx4j-tools.jar is under the directory of $kAFKA_HOME/libs 
3. start broker, and check if http://$broker_ip:$MX4J_PORT works.

### Committer Checklist (excluded from commit message)
